sagemaker-debugger/build_your_own_container_with_debugger/docker/tf_keras_resnet_byoc.py

```diff
@@ -0,0 +1,110 @@
+"""
+This script is a ResNet training script which uses Tensorflow's Keras interface, and acts as an
+example of how to use SageMaker Debugger if you use either your own custom container to run in SageMaker, or your own script outside SageMaker.
+It has been orchestrated with SageMaker Debugger hooks to allow saving tensors during training.
+These hooks have been instrumented to read from json configuration that SageMaker will put in the training container.
```

Comment:
Change to: "These hooks have been instrumented to read from a JSON configuration that SageMaker puts in the training container."